webpack.config.js

```diff
@@ -8,15 +8,15 @@ module.exports = {
     path: path.join(__dirname, '/dist'),
     filename: 'bundle.js',
   },
+  resolve: {
+    extensions: ['', '.jsx', '.js', '.json']
+  },
```

Comment:
just another webpack thing. it probably does these javascript type extensions by default, but i always like to define them.